### ISSUE TITLE
Map Downloader

### ISSUE THREAD

**Issue by LiTTLEUNMAN** [ionar2/salhack#201]
A module that saves the currently held map to a folder as a PNG similar to the ForgeHax module but actually functional. This would be useful for adding to map art archives.
[https://github.com/fr1kin/ForgeHax/wiki/MapDownloader](url)
[https://github.com/fr1kin/ForgeHax/blob/master/src/main/java/com/matt/forgehax/mods/MapDownloader.java](url)

**Comment by dominikaaaa** [comment]
What's wrong with the forgehax one?

**Comment by LiTTLEUNMAN** [comment]
The forgehax one just saves a 128x128 px black square and not the actual map image

**Comment by dominikaaaa** [comment]
That sounds like a you problem, it works fine for me and everyone else.

**Comment by LiTTLEUNMAN** [comment]
Yeah, I've seen it work fine for some people and not work at all for others
